Admin/BaseMediaAdmin.php

```diff
@@ -115,7 +115,7 @@ public function getNewInstance()
                     );
                 } else {
                     $media->setProviderName(
-                        $this->getRequest()->get($uniqid['providerName'])
+                        $this->getRequest()->get($uniqid)['providerName']
```

Comment:
Function array dereferencing does not work on old versions of php, you will have to change that (that's why the build fails)